src/main/java/bio/terra/cli/apps/DockerUtils.java

```diff
@@ -0,0 +1,244 @@
+package bio.terra.cli.apps;
+
+import bio.terra.cli.command.exception.SystemException;
+import bio.terra.cli.command.exception.UserActionableException;
+import bio.terra.cli.context.utils.Printer;
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.async.ResultCallback;
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import com.github.dockerjava.api.command.WaitContainerResultCallback;
+import com.github.dockerjava.api.exception.NotFoundException;
+import com.github.dockerjava.api.model.Bind;
+import com.github.dockerjava.api.model.Frame;
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.api.model.Volume;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.core.DockerClientImpl;
+import com.github.dockerjava.httpclient5.ApacheDockerHttpClient;
+import com.github.dockerjava.transport.DockerHttpClient;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/** This class provides utility methods for running Docker containers. */
+public class DockerUtils {
+  private final DockerClient dockerClient;
+
+  public DockerUtils() {
+    this.dockerClient = DockerUtils.buildDockerClient();
```

Comment:
Little surprising to have a `Utils` class not be static. Maybe just have our own `DockerClient` class or `DockerClientWrapper`?

Reply:
Changed to `DockerClientWrapper`.